.ci/msys_test.bat

```diff
@@ -0,0 +1,7 @@
+:: FIXME(equalsraf) currently not all tests work on Windows
+:: for now just run what we can
+mingw32-make functionaltest TEST_FILE=test\functional\eval\printf_spec.lua VERBOSE=1 || goto :error
```

Comment:
Problem with using pending() like that is that for it would likely end up being used in most tests. And the extra call to pending() seems expensive.

> That's the way it supposed to be supposed to be (and it works for me locally on linux). 

I see whats happening now. In tests that do the early checks for pending, like python3_spec the tag is only matched after evaluating the pending check - turns out the check itself causes it to fail in windows :D. There seem to be other cases where early evaluation of the test specs causes failures, but its hard to debug because there is no output.